editoast/fga/src/client.rs

```diff
@@ -43,8 +43,8 @@ pub const DEFAULT_OPENFGA_MAX_TUPLES_PER_WRITE: u64 = 100;
 
 #[derive(Debug, Clone)]
 pub struct Client {
-    store: Store,
-    authorization_model_id: Option<String>,
+    pub store: Store,
+    pub authorization_model_id: Option<String>,
```

Comment:
Probably doesn't need to be pub. Most of the occurrences are for logging, but they're already provided to the spans of instrumented functions in the Client (even though span and logs aren't the same thing admittedly). Here's my proposition:

* make authorization_model_id private and get rid of the logging line
* provide a `pub fn store(&self) -> &Store` to prevent mutability that the Client doesn't support (nothing works if I mutate the store and forget to change the model ID manually as well).